src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix that filters flags
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
+    {
+        $constants = get_defined_constants();
+        foreach ($constants as $c => $v) {
+            if ('' !== $prefix && 0 !== strpos($c, $prefix) || ($value & $v) !== $v) {
+                unset($constants[$c]);
+            }
+        }
+
+        asort($constants);
+        $name = implode(' | ', array_flip($constants)) ?: 0;
```

Comment:
yeah, but it is not displayed by default. So using a ConstStub for flags is valid only if the textual representation represents **all** bits of the flag.